cdc/model/owner.go

```diff
@@ -67,6 +68,14 @@ func (t AdminJobType) IsStopState() bool {
 	return false
 }
 
+// DDLJobEntry is the DDL job entry.
+type DDLJobEntry struct {
+	Job    *timodel.Job
+	OpType OpType
+	CRTs   uint64
+	Err    error
```

Comment:
```suggestion
	Error    error
```

To keep the same style with other structs in this package.